test/basic.html

```diff
@@ -173,6 +202,45 @@
           assert.isTrue(smallRect.height >= 1 * parseFloat(smallStyle.fontSize));
         });
       });
+
+      suite('ink size', function() {
+        var checkboxes;
+
+        setup(function() {
+          checkboxes = fixture('WithDifferentSizes2');
+        });
+
+        test('`--paper-checkbox-ink-size` sets the ink size', function() {
```

Comment:
(from offline) The `var(--calculated-paper-checkbox-size)` -> `var(--calculated-paper-checkbox-ink-size)` change only affects the ripple.